tests/slepos/build_images_creator.pm

```diff
@@ -0,0 +1,254 @@
+# SUSE's openQA tests
+#
+# Copyright © 2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Build SLEPOS images using image creator
+# Maintainer: Pavel Sladek <psladek@suse.cz>
+
+use base "basetest";
+use strict;
+use warnings;
+use testapi;
+use utils;
+
+
+
+my %template_done;
+
+sub build_image {
+
+    my ($image, $template, $template_idx, $packages_ar) = @_;
+    my $img_suffix = 'tar.bz2';
+
+    $image =~ /^(.*)-([^-]*)$/;
+    my $imagebase = $1;
+    my $version   = $2;
+    bmwqemu::diag("image '$image' will be built using image creator:");
+
+    #add virtual drivers unless already done
+    if (!$template_done{$template}) {
+        select_console('root-console');
+        wait_still_screen;
+        save_screenshot;
+
+        prepare_template($template);
+        $template_done{$template} = 1;
+
+        select_console('x11');
+        wait_still_screen;
+    }
+
+    # assume image creator is started, and on the add image screen
+    #add image
+    mouse_hide;
+    save_screenshot;
+    assert_and_click('image-creator-addimage');
+    send_key('alt-s');
+    save_screenshot;
+    assert_and_click('image-creator-list-templates');
+    #select image template
+    save_screenshot;
+    send_key('end');
+    for (my $i = 0; $i < $template_idx; $i++) {
+        send_key('up');
+    }
+    wait_still_screen;
+    save_screenshot;
+    send_key('ret');
+
+    assert_and_click('image-creator-imagename');
+    type_string($imagebase);
+    save_screenshot;
+    #todo: uncheck 32bit for (get_var('VERSION') =~ /^12/) {
+    assert_and_click('image-creator-next1');
+    if (check_screen('import-untrusted-gpg-key')) {
```

Comment:
please try

```perl
assert_screen [qw(import-untrusted-gpg-key image-creator-image-configuration)];
send_key 'alt-i' if match_has_tag 'import-untrusted-key';
```

This can save the 30s waiting time in case of no untrusted key